include/ssl_compat.h

```diff
@@ -62,6 +62,13 @@
 #endif /* HAVE_OPENSSL11 */
 #endif
 
+#ifdef LIBRESSL_VERSION_NUMBER
+#define EVP_MD_CTX_SIZE			64
+#define EVP_CIPHER_CTX_SIZE             192
+#define EVP_MD_CTX_init(X) do { memset((X), 0, EVP_MD_CTX_SIZE); EVP_MD_CTX_reset(X); } while(0)
+#define EVP_CIPHER_CTX_init(X) do { memset((X), 0, EVP_CIPHER_CTX_SIZE); EVP_CIPHER_CTX_reset(X); } while(0)
+#else
```

Comment:
To avoid another nesting and have an `#endif` here, afaik  only a `#ifndef EVP_CIPHER_CTX_SIZE` around the `#define EVP_CIPHER_CTX_SIZE` below is needed.